### PR TITLE
add workflow to read-me

### PR DESCRIPTION
**Pull request by lucafurrer**
I added a section workflow into the read-me.
A question still around from my side was if there is a need for a standard naming conventions for branches.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,9 +25,15 @@ Docker containers will start and the services will be available under the these
...
+| UI      | <https://app.cube-creator.lndo.site/> |
+| Fuseki  | <https://db.cube-creator.lndo.site/>  |
 
 Lando uses its own Certificate Authority and it won't be trusted by your system.
-To trust the CA, follow the steps on https://docs.lando.dev/config/security.html#trusting-the-ca
+To trust the CA, follow the steps on <https://docs.lando.dev/config/security.html#trusting-the-ca>
+
+## Workflow
+
+This repository is using the [GitHub-Flow](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/github-flow). Hence all changes should integrated using pull requests.
```
Comment (vhf) [inline README.md:37]:
```suggestion
This repository is using the [GitHub-Flow](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/github-flow). Hence all changes should be integrated using pull requests.
```

**Review by vhf — APPROVED** [review]
👍 , could you please also add a reference to https://www.conventionalcommits.org/en/v1.0.0/#summary ?

**Comment by tpluscode** [comment]
@vhf actually no decision regarding conventional commits has been made so far.

**Comment by vhf** [comment]
IMO documenting what we have been doing so far would make sense

**Comment by lucafurrer** [comment]
> @vhf actually no decision regarding conventional commits has been made so far.

@tpluscode @vhf I am open to include conventional commits. I can add it and we change it, if needed or we can have a discussion first. as you prefer.

**Comment by vhf** [comment]
Please add it then, glancing at the commit log it's a convention we've been (more or less) applying so far, adding a reference to its source makes sense. No need to 100% commit to it so let's stay flexible and only have a note saying something along the lines of "Commit messages usually follow …". That way we won't have issues with the occasional automatically-generated commit messages that won't fit.

**Pull request merged**